config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml

```diff
@@ -126,6 +126,39 @@ periodics:
       - --timeout=120m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20190529-49b63a5-master
 
+- name: ci-kubernetes-e2e-windows-gce-k8sstable1
+  decorate: true
+  extra_refs:
+  - org: yujuhong
```

Comment:
you have your own org?

Reply:
everyone has its own org!

Reply:
we should talk about moving this to kubernetes-sigs

Reply:
Agree. This is in our backlog. We could try moving it to https://github.com/kubernetes-sigs/windows-testing/